js/huobi.js

```diff
@@ -30,7 +30,9 @@ module.exports = class huobi extends Exchange {
                 'CORS': undefined,
                 'createOrder': true,
                 'fetchBalance': true,
+                'fetchBorrowRate': true,
                 'fetchBorrowRates': true,
+                'fetchBorrowRatesPerSymbol': true,
```

Comment:
I would not add this to the set of unified methods if it is not unified across the other exchanges yet.